XR_APILAYER_NOVENDOR_toolkit/layer.cpp

```diff
@@ -1366,6 +1366,14 @@ namespace {
                 m_posesForFrame[0].pose = views[0].pose;
                 m_posesForFrame[1].pose = views[1].pose;
 
+                // No inter eye rotation workaround
+                if (m_configManager->getValue(config::SettingNoInterEyeRotation)) {
+                    views[0].pose.orientation.w = views[1].pose.orientation.w;
+                    views[0].pose.orientation.x = views[1].pose.orientation.x;
+                    views[0].pose.orientation.y = views[1].pose.orientation.y;
+                    views[0].pose.orientation.z = views[1].pose.orientation.z;
```

Comment:
Copying the quaternion isn't the right thing to do. This might look OK with very low canting like WMR. But with actual canted displays like varjo and Pimax, this will introduce a warping effect. Better to set the quaternions for both eyes to xr::Quaternion::Identity()

Reply:
Do you mean directly setting it here to xr::Quaternion::Identity()? Tried that and didn't work. Or do you have to transform it to some space, apply the identity and then transform back (like you indicated in your discord post referring to the pimax come)? I also played around with finer canting values (your code after this) but this didn't produce the right results

Reply:
Hmmm OK, then I have no idea what is wrong with Fallout 4. I've looked again at the WMR for SteamVR code, and it does what you did (different way, same result). I however only foresee that it will work on WMR. So we should hide the option on other devices.

You can take a look at how to create menu entries that are only enabled on certain configuration, look for Pimax WFOV hack for example. That should do the trick. Also be sure to only apply the transform "if WMR and if no_inter_eye_rotation" and not just "if no_inter_eye_rotation". This is because certain users will switch headsets once in a while, and hiding the option is not enough, it needs to be disabled entirely.